ci/cron/compare-perf.sh

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+BASELINE=$1
+
+measure() {
+    local treeish=$1
+    local out=$(mktemp -d)/out.json
+    git checkout $treeish >&2
+    bazel run daml-lf/scenario-interpreter:scenario-perf -- -rf json -rff $out >&2
+    cat $out | jq '.[0].primaryMetric.score'
+}
+
+main() {
+  local current=$(git rev-parse HEAD)
+
+  local baseline_perf=$(measure $BASELINE)
+  local current_perf=$(measure $current)
+
+  git checkout $current >&2
+
+  local factor=$(echo "$current_perf / $baseline_perf" | bc -l)
+  local speedup=$(printf "%.1f" $(echo "($factor - 1) * 100" | bc -l))
+  local kpi_completion=$(printf "%.2f" $(echo "100 / (l(5) / l($factor))" | bc -l))
```

Comment:
Nit: This isn’t actually true. Our KPI is for the performance of a specific codebase not this benchmark.

Reply:
I guess I was assuming this benchmark had been designed to be a somewhat accurate reflexion, as it seems to be the one used to judge individual performance PRs. I'll update the wording to be more prudent.